src/cache.rs

```diff
@@ -13,7 +19,7 @@ pub struct Cache {
 
 impl Cache {
     pub fn new() -> Result<Self, Box<dyn Error>> {
-        let path = xdg::BaseDirectories::new()?.place_state_file("comma-choices")?;
+        let path = xdg::BaseDirectories::new()?.place_state_file("comma/choices")?;
```

Comment:
I am changing the state file here for 2 reasons:
- One to keep it in a more standardized place (in `$XDG_STATE_HOME/comma/choices` instead of `$XDG_STATE_HOME/comma-choices`)
- Because the format changed and I don't want to write a migration